presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushProjectionThroughExchange.java

```diff
@@ -35,6 +35,37 @@
 import java.util.Map;
```

Comment:
Instead of repeating the commit title of the other commit with the "(changes)" qualifier, describe what the changes are.